kafka/consumer_test.go

```diff
@@ -429,3 +430,223 @@ func TestConsumerLog(t *testing.T) {
 		}
 	}
 }
+
+var assignedEvents int32
+var revokedEvents int32
+
+func rebalanceCallback(c *Consumer, event Event) error {
+	switch ev := event.(type) {
+	case AssignedPartitions:
+		atomic.AddInt32(&assignedEvents, 1)
+		fmt.Fprintf(os.Stderr,
```

Comment:
nit: Tests should use t.Logf if possible, but that might be tricky here in the callback.

Reply:
Yeah, is there any good way to use `t *testing.T`  in the callback?

Reply:
I don't know, really. Go 🤷